pkg/volume/csi/csi_mounter.go

```diff
@@ -252,10 +252,18 @@ func (c *csiMountMgr) SetUpAt(dir string, fsGroup *int64) error {
 }
 
 func (c *csiMountMgr) GetAttributes() volume.Attributes {
+	mounter := c.plugin.host.GetMounter(c.plugin.GetPluginName())
+	path := c.GetPath()
+	supportSelinux, err := mounter.GetSELinuxSupport(path)
+	if err != nil {
+		glog.V(2).Infof("Error checking for SELinux support: %s", err)
```

Comment:
To be consistent with other logging in csi `glog.V(2).Info(log("Error checking for SELinux support: %s", err))`

Reply:
Fixed